bot/resources/tags/json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
+A JSON may never be completely empty and must always at least have one of the following items.
+
+```
+object
+array
+string
+number
+"true"
+"false"
+"null"
+```
```

Comment:
I should've been more unequivocal. Listing out all possibilities isn't really beneficial and will likely serve to confuse people. 99% of the time people will either want an object or an array. Therefore, this tag should only mention those two cases e.g. "You probably want to use an array or object, blah blah blah..."